src/components/KonnectorTile.jsx

```diff
@@ -2,26 +2,60 @@ import React from 'react'
 import { connect } from 'react-redux'
 import { translate } from 'cozy-ui/react/I18n'
 
+import { NavLink, withRouter } from 'react-router-dom'
+
+import { getKonnectorIcon } from '../lib/icons'
 import { getKonnectorTriggersCount } from '../reducers'
 
-import Tile from './Tile'
+const KonnectorTile = ({ footer, konnector, route, t }) => {
+  const categories = konnector.categories
+    ? konnector.categories.map(c => t(`category.${c}`))
+    : []
+  const subtitle = categories.join(', ')
+  return (
+    <NavLink className="item-wrapper" to={route}>
+      <header className="item-header">
+        <img
+          className="item-icon"
+          alt={t('connector.logo.alt', { name })}
+          src={getKonnectorIcon(konnector)}
+        />
+      </header>
+      <h3 className="item-title">{name}</h3>
+      {subtitle && <p className="item-subtitle">{subtitle}</p>}
+      {footer}
+    </NavLink>
+  )
+}
 
-const KonnectorTile = props => (
-  <Tile route={props.route} subtitle={props.subtitle} {...props} />
+const svgIcon = name => (
```

Comment:
There already is an `<Icon />` component in cozy-ui for this exact same use case, did you try it?

Reply:
I gave it a quick try and it did not work immediately. So maybe it could be a nice improvement for later.